app/assets/stylesheets/pages/_moodcard.scss

```
@@ -0,0 +1,6 @@
+.moodcard {
+  width:100%;
+  height:208px;
+  border-radius: 20%;
```

I think setting border-radius to 20% here is a mistake since percentage yield strange results on rectangles.
In addition you then override it in the divs by setting it to 20px in each of them. I suggest a consolidation by setting it to 20px here, and removing the overrides from the DIVs where you apply this class.

```suggestion
  border-radius: 20px;
```